src/jira/client/jira-api-client.test.ts

```diff
@@ -0,0 +1,108 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import { getLogger } from "config/logger";
+import { JiraClient } from "./jira-api-client";
+import { DatabaseStateCreator } from "test/utils/database-state-creator";
+
+
```

Comment:
Pulled the models/jiraclient tests across and just tweaked the before each to fit the new create style, otherwise same tests